CHANGELOG

```diff
@@ -7,6 +7,7 @@
  * Change the compilation of `for` loops to throw an exception when a `loop.*` variable is not defined
  * Make `Environment::getGlobals()` private
  * Drop support for PHP < 8.2
+ * Unnecessary escape characters are no longer ignored
```

Comment:
I'm not sure I understand the meaning. Can you clarify?

Reply:
I don't know how to explain it better.

Doing `App\Test` tries to escape T which is unnecessary. In v3 this escape `\` is ignored and reported as deprecation. In v4 it's no longer ignored, and results in a literal `\`. That means that `App\Test` will show up as `App\Test` as you expect.

Reply:
Maybe something like this, as it will be hard to explain simply with a short sentence  ?

```
 * Change tokenisation of escape sequence in strings

 Before::  
  // ... 
 {# foo equals "FooBar" #}
 
After:
 // 
 {# foo equals "Foo\Bar" #}
```

As example I would probably use a function call (instead of a print expression for instance), to support the idea it's a _lexer thing_ and thus is not related to how context vars are printed.. wdyt ?

Reply:
Or maybe just "Escape characters are no longer ignored".